be/src/main/java/com/ijava/todolist/card/repository/JdbcCardRepository.java

```diff
@@ -19,36 +22,97 @@ public class JdbcCardRepository implements CardRepository {
 
     private final JdbcTemplate jdbcTemplate;
     private final SimpleJdbcInsert simpleJdbcInsert;
+    private final NamedParameterJdbcTemplate namedParameterJdbcTemplate;
 
     public JdbcCardRepository(DataSource dataSource) {
         this.jdbcTemplate = new JdbcTemplate(dataSource);
         this.simpleJdbcInsert = new SimpleJdbcInsert(dataSource);
+        this.namedParameterJdbcTemplate = new NamedParameterJdbcTemplate(dataSource);
+
         simpleJdbcInsert.withTableName("card").usingGeneratedKeyColumns("id");
     }
 
     @Override
     public Card save(Card card) {
+        if (card.getId() == null) {
+            return insertCard(card);
+        }
+
+        return updateCard(card);
```

Comment:
오 ㅎㅎ save()의 기능을 잘 구현하셨군요 👍

nit;
Card 내에 메서드명을 IsSave()로 구현하면 의미가 명확히 파악될 수 있겠네요 : )

```suggestion
        if (isSave()) {
            return insertCard(card);
        }

        return updateCard(card);
```